site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,119 @@
+using the `navigator.storage.estimate()` method. Storage can also be evicted under heavy memory
+pressure, although this is rare. To avoid this:
+
+- Request the `unlimitedStorage` permission, which affects both extension and web storage APIs and
+exempts extensions from both quota restrictions and eviction.
+- Use [`navigator.storage.persist()`][storage-persist] for protection against eviction.
+
+Extension storage is shared across the extension's origin, such as the associated service worker,
+any extension pages including popups and the side panel, and offscreen documents. In content
+scripts, calling web storage APIs accesses data from the origin the content script is injected on
+and not the extension.
+
+### Access in service workers {: #storage-in-service-workers }
+
+Some web storage APIs like [IndexedDB][indexeddb] are accessible in service workers. However,
+[Local Storage][local-storage] and [Session Storage][session-storage] are not.
+
+Instead, use an [offscreen document][offscreen]. For example, you could use the following to migrate
+data when updating from Manifest V2 to Manifest V3:
+
+1. In the extension service worker check [`chrome.storage`][chrome-storage-api] for your data.
+1. If your data isn't found, [create][create-offscreen] an offscreen document and call
+[`sendMessage()`][send-message] to start a conversion routine. Read the data from the web storage
+APIs and use message passing to return it to the service worker.
+1. In the service worker, write the data to a new location.
+
+### Partitioning {: #storage-partitioning }
+
+Partitioning is where keys are introduced for stored data to limit where it can be accessed.
+
+To prevent certain types of cross-site tracking, [storage partitioning][storage-partitioning]
+introduces changes to how partitioning keys are defined. In practice, this means that if site A
+embeds an iframe containing site B, site B will not be able to access the same storage it would
+usually have when navigated to directly.
+
+To mitigate the impact of this in extensions, two key exemptions apply:
+
+- If a page with the `chrome-extension://` scheme is embedded in any site, storage partitioning will
+not apply, and the extension will have access to its top-level partition.
+- If a page with the `chrome-extension://` scheme includes an iframe, and the extension has
+[host permissions][declare-permissions] for the site it is embedding, that site will also have
+access to its top-level partition.
+
+## Cookies {: #cookies }
+
+Cookies provide a way to store key/value pairs associated with a specific domain and path. They have
+limited value in extensions but understanding their behavior can be useful if you have a specific
+use case or have bundled a third-party script which uses them in its implementation.
+
+### Secure cookies {: #secure-cookies }
+
+The [`Secure`][cookies-restrict-access] cookie attribute is only supported for the `https://`
```

Comment:
I didn't realise this. It raises an interesting additional point, presumably that means `chrome-extension://` URLs also cannot set a `SameSite=None`, i.e. a third-party cookie, as this also requires the `Secure` attribute. If that's the case, `Partitioned` would simply be a continuation of this - you can't have the entire super-set of cross-site cookie options, `Partitioned` is a specific option within that.

Reply:
Good call! I added a note 😃